src/templates/static/metatrader/download.jsx

```diff
@@ -1,64 +1,111 @@
+                {it.L('MT5 for Linux')}
+            </h1>
+            <p>
+                {it.L(
+                    'Download MT5 for your desktop or laptop to access the powerful tools and features enjoyed by millions of traders.'
+                )}
+            </p>
             <div className='gr-row'>
-                <DownloadBlock
+                <DesktopDownloadBlock
+                    id='windows'
                     title={it.L('Windows')}
-                    desc={it.L('Windows 7 or later')}
                     image='windows'
                     href='https://s3.amazonaws.com/binary-mt5/binarycom_mt5.exe'
                 />
-                <DownloadBlock
+                <DesktopDownloadBlock
+                    id='macos'
                     title={it.L('MacOS')}
-                    desc={it.L('All versions of MacOS')}
                     image='apple'
                     href='https://s3.amazonaws.com/binary-mt5/binary-mt5.dmg'
                     download='true'
                 />
-                <DownloadBlock
+                <DesktopDownloadBlock
+                    id='linux'
                     title={it.L('Linux')}
-                    desc={it.L('All versions of Linux')}
                     image='linux'
                     href='https://www.metatrader5.com/en/terminal/help/start_advanced/install_linux'
                     target='_blank'
                 />
             </div>
-            <p className='foot-note gr-padding-20'>{it.L('The MetaTrader platform will no longer support Windows XP, Windows 2003, and Windows Vista after 01 October, 2017.') }</p>
+            <p className='alternative-download-description invisible' id='macos-alternative-description'>
+                {it.L(
+                    'Looking for [_1] or [_2] Desktop apps?',
+                    `<a data-type='alt-link' data-target='windows' id='enable-windows-alt-link'>${it.L('Windows')}</a>`,
+                    `<a data-type='alt-link' data-target='linux' id='enable-linux-alt-link'>${it.L('Linux')}</a>`
+                )}
+            </p>
+            <p className='alternative-download-description invisible' id='windows-alternative-description'>
+                {it.L(
+                    'Looking for [_1] or [_2] Desktop apps?',
+                    `<a data-type='alt-link' data-target='macos' id='enable-mac-alt-link'>${it.L('Mac')}</a>`,
+                    `<a data-type='alt-link' data-target='linux' id='enable-linux-alt-link'>${it.L('Linux')}</a>`
+                )}
+            </p>
+            <p className='alternative-download-description invisible' id='linux-alternative-description'>
+                {it.L(
+                    'Looking for [_1] or [_2] Desktop apps?',
+                    `<a data-type='alt-link' data-target='windows' id='enable-windows-alt-link'>${it.L('Windows')}</a>`,
+                    `<a data-type='alt-link' data-target='macos' id='enable-mac-alt-link'>${it.L('Mac')}</a>`
```

Comment:
I would suggest to store each anchor as it seems to be reusable

Reply:
I couldn't do this. By refactoring the string part only, `it.L` will be undefined, and changing it to `localize` function would result in `ReferenceError: Element is not defined`.

I'm not sure, but by removing localize import the error goes away.

```
at Object.<anonymous> (src/javascript/_common/lib/polyfills/element.matches.js:1:63)
```

Reply:
look at something like this https://github.com/binary-com/binary-static/blob/master/src/templates/app/cashier/payment_methods.jsx#L78, we declare the variable outside of the return function and then use it inside, localization works fine